cmd/security-bootstrapper/entrypoint-scripts/vault_wait_install.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#  ----------------------------------------------------------------------------------
+
+# This is customized entrypoint script for Vault.
+# In particular, it waits for the BootstrapPort ready to roll
+
+set -e
+
+# env settings
+. /edgex-init/.env-vault
+
+echo "Script for waiting security bootstrapping installation"
+
+DEFAULT_VAULT_LOCAL_CONFIG='
+listener "tcp" { 
+              address = "edgex-vault:8200" 
+              tls_disable = "1" 
+              cluster_address = "edgex-vault:8201" 
+          } 
+          backend "file" {
+              path = "/vault/file"
+          } 
+          default_lease_ttl = "168h" 
+          max_lease_ttl = "720h"
+'
+
+VAULT_LOCAL_CONFIG=${VAULT_LOCAL_CONFIG:-$DEFAULT_VAULT_LOCAL_CONFIG}
+
+export VAULT_LOCAL_CONFIG
+
+echo "$(date) VAULT_LOCAL_CONFIG: ${VAULT_LOCAL_CONFIG}"
+
+if [ "$1" = 'server' ]; then
```

Comment:
see ADR, vault also needs to wait on bootstrap port

Reply:
Ok, now I see what `WAIT_TCP_PORT` is. Since bootstrapper has multiple ports to be waiting on a better name would be `WAIT_BOOSTRAPPER_STARTED_PORT` to be explicit what it is waiting on.

Reply:
sure, i am fine with that